### PR TITLE
chore(localdebug): use vscode config instead of  localsettings for skipNgrok

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Detail: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12863558/

### CODE REVIEW

File: packages/vscode-extension/src/debug/depsChecker/vscodeHelper.ts [hunk 1]
```diff
@@ -28,6 +28,10 @@ class VSCodeHelper {
     return this.checkerEnabled("prerequisiteCheck.node");
   }
 
+  public isNgrokEnabled(): boolean {
```
Comment (XiaofuHuang) [inline packages/vscode-extension/src/debug/depsChecker/vscodeHelper.ts:31]: TODO: move isXXXEnabled out of depsChecker

File: packages/api/src/types.ts [hunk 2]
```diff
@@ -231,6 +231,7 @@ export interface Inputs extends Json {
   env?: string;
   projectId?: string;
   existingResources?: string[];
+  skipNgrok?: boolean;
```
Comment (swatDong) [inline packages/api/src/types.ts:234]: Make this extensible? E.g., `depsInfo` or `checkerInfo` or `ignoreCheckers`, etc.

**Review by swatDong — APPROVED** [review]

**Comment by kuojianlu** [comment]
Remember to update the help doc as well: https://github.com/OfficeDev/TeamsFx/blob/dev/docs/fx-core/localdebug-help.md#mitigation-1.

**Comment by XiaofuHuang** [comment]
> Remember to update the help doc as well: https://github.com/OfficeDev/TeamsFx/blob/dev/docs/fx-core/localdebug-help.md#mitigation-1.

Thanks for reminder!

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by meifans — APPROVED** [review]
LGTM

**Pull request merged**